png_benchmark.py

```diff
@@ -0,0 +1,152 @@
+                test_result = {
+                    "time_ns": time_ns,
+                    "size_B": size_B,
+                    "size_ratio_compressed_to_original_%": compression_ratio
+                }
+                results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"]["frame_data"].append(test_result)
+
+                # Save one image (this one has 2 landing pads in it) for reference
+                if frame_index == FRAME_TO_SAVE:
+                    img.save(pathlib.Path(OUTPUT_PATH, f"ct{compress_type}_cl{compress_level}.png"), "PNG", compress_level=compress_level, compress_type=compress_type)
+            
+            # Save average test results
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_TIME_MS] = max_time_ns / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_TIME_MS] = min_time_ns / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_TIME_MS] = total_time_ns / FRAME_COUNT / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_SIZE_B] = max_size_B
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_SIZE_B] = min_size_B
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_SIZE_B] = total_size_B / FRAME_COUNT
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_RATIO_COMPRESSED_TO_ORIGINAL] = max_compression_ratio
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_RATIO_COMPRESSED_TO_ORIGINAL] = min_compression_ratio
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_RATIO_COMPRESSED_TO_ORIGINAL] = total_compression_ratio / FRAME_COUNT
+            print(f"Compress level {compress_level} complete")
+    
+    print()
+    print("-------------------TEST COMPLETED------------------")
+    print()
+    
+    # Saving full results
+    with open(pathlib.Path(OUTPUT_PATH, "results.json"), 'w', encoding="utf-8") as file:
+        file.write(json.dumps(results, indent=2))
+    
+    # Saving shortcut results without frame data (for more human readability)
+    with open(pathlib.Path(OUTPUT_PATH, "summary.csv"), 'w', encoding="utf-8") as file:
+        file.write("Compression Type,Compression Level,Max Time (ms),Min Time (ms),Avg Time (ms),Max Size (B),Min Size (B),Avg Size (B),Max Size Ratio (compressed to original in %),Min Size Ratio (compressed to original in %),Avg Size Ratio (compressed to original in %)\n")
+        for compress_type in COMPRESS_TYPES:
+            for compress_level in range(INITIAL_COMPRESS_LEVEL, MAX_COMPRESS_LEVEL + 1):
+                current_result = results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"]
```

Comment:
Line too long, maximum is 100 characters. Use intermediate string variables and/or intermediate dictionary.